Kudu.Core/Jobs/ContinuousJobsManager.cs

```diff
@@ -57,6 +62,39 @@ public void EnableJob(string jobName)
             continuousJobRunner.EnableJob();
         }
 
+        public async Task<HttpResponseMessage> RequestPassthrough(string jobName, string path, HttpRequestMessage request)
+        {
+            using (var client = new HttpClient())
+            {
+                var forwardRequest = GetWebJobForwardRequest(jobName, path, request);
+                return await client.SendAsync(forwardRequest);
+            }
+        }
+
+        private HttpRequestMessage GetWebJobForwardRequest(string jobName, string path, HttpRequestMessage original)
```

Comment:
nit: GetForwardRequest? (basically I'm questining the "WebJob" name prefix you're using everywhere - seems unecessary)